### PR TITLE
Improve ~remote~ _playbook_ instructions.

### PR DESCRIPTION
**Pull request by rosiel**
## Purpose / why

The instructions are outdated! We no longer need karaf or so many fedora uris.

## What changes were made?

* removed deprecated options
* updated the sample new folder name from `example` to `production` to give it more realism and also to match the command at the bottom

## Verification

Can you use this to provision a ubuntu 20.04, and get all the expected behaviours?

## Interested Parties

@alxp @whikloj 

### Pull-request Reviewer
Pull-request reviewer should ensure the following:

* [ ] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [ ] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [ ] Are the changes accurate, useful, free of typos, etc?

### Person Merging
The person merging should ensure the following:

* [ ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [ ] If pages are renamed or removed, have all internal links to those pages been fixed?
* [ ] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

**Comment by rosiel** [comment]
Current issues:

1. Can't _index_ into Fedora. After playbook running, there are ~32 messages dequeued from the islandora-indexing-fcrepo-content, but once i expose the Fedora interface (see below) i see none of them are in Fedora. The fcrepo/rest interface shows an empty repository with no "children". 

The method I used to open Fedora was the "alternatively"  `fcrepo.xml` file approach here: https://wiki.lyrasis.org/display/FEDORA6x/Disable+Authentication+and+Authorization. Once this has been done, i see new "children" in Fedora when i add content in Drupal. This definitely isn't a step i need to take in Vagrant (for indexing OR for access at :8080), but I'm not sure why. By the time I checked (the playbook had finished) the admin user had fedoraAdmin role.

Here's a sample of (one node's worth of) [errors in milliner log](https://gist.github.com/rosiel/72bec715beb81a1660ccab218f95429d). 

2. Known issue, affects Vagrant too: We're putting messages to `islandora-indexing-fcrepo-file-external` but nothing is reading them: the "Number of Consumers" is 0. 

3. FITS isn't  ... working? Using both the automatic Contexts and the Action to create FITS derivatives, I get nothing back. The messages are being dequeued from the ActiveMQ. There's an auth error in the alpaca logs. See [error log snippet](https://gist.github.com/rosiel/9f16d19e8e843cca902f6149ea5e726d).

**Comment by whikloj** [comment]
I don't see any lines marked with `ERROR` in the milliner log snippet you referenced. Is there more after? Also it would be good to see what the Fedora log is saying too.

**Comment by rosiel** [comment]
I tried a fresh server and the milliner problems didn't happen (Also, wow I think I failed to copy the wrong section of those logs!)

Fits is kind of a different thing, so i'm investigating that separately, but it probably doesn't have to do with the remote deployment instructions.

File: docs/installation/playbook.md [hunk 1]
```diff
@@ -96,78 +96,29 @@ your own values.
 #### group_vars/all/passwords.yml
 
 ```yml
+# Drupal
 drupal_db_password: changeme
 drupal_account_pass: changeme
-islandora_db_root_password: changeme
```
Comment (rosiel) [inline docs/installation/playbook.md:101]: don't worry the rest of the passwords are there on the right... just bumped down in the diff.

File: docs/installation/playbook.md [hunk 2]
```diff
@@ -96,79 +96,24 @@ your own values.
...
-```
-
-#### group_vars/tomcat.yml
+# Cantaloupe
+cantaloupe_admin_password: changeme
 
-```yml
-fcrepo_allowed_external_content:
-  - http://example.org/
-cantaloupe_HttpResolver_BasicLookupStrategy_url_prefix: http://example.org/
```
Comment (rosiel) [inline docs/installation/playbook.md:171]: This was breaking cantaloupe, hat-tip to @whikloj for helping find the solution.

**Comment by ysuarez** [comment]
Is this ready for merging, do we need a final person besides Rosie to give it a final "blessing" before we merge?

**Comment by rosiel** [comment]
Sorry for the last minute changes - we just updated the interface for using the Playbook so I put the new info here.

**Comment by rosiel** [comment]
I've deleted the sneaky files that I added by mistake (thanks whoever pointed it out!).

**Pull request merged**